data/json/monsters.json

```diff
@@ -1844,6 +1848,36 @@
     "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_screecher",
+    "name": "screecher zombie",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"magenta_cyan",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":15,
+    "aggression":100,
+    "morale":100,
+    "speed":100,
+    "melee_skill":5,
+    "melee_dice":3,
+    "melee_dice_sides":4,
+    "melee_cut":5,
+    "dodge":2,
+    "armor_bash":0,
+    "armor_cut":0,
+    "vision_day":50,
+    "vision_night":8,
+    "death_drops": "default_zombie_death_drops",
+    "luminance":0,
```

Comment:
Nit, if the attribute is at the default value, leave it out, no need to have hundreds of monster entries all asserting that the monsters aren't glowing.